torjail

```diff
@@ -373,13 +373,14 @@ if [ $? -ne 0 ]; then
 
   # REJECT all traffic coming from torjail
   # this is needed to avoid reaching other interfaces
-  iptables -A INPUT -i in-$NAME -p udp --destination $IPHOST --dport 5354 -j ACCEPT
-  iptables -A INPUT -i in-$NAME -p tcp --destination $IPHOST --dport 9040 -j ACCEPT
-  iptables -A INPUT -i in-$NAME -p udp --destination $IPHOST --dport 9040 -j ACCEPT
+  iptables -I INPUT -i in-$NAME -p udp --destination $IPHOST --dport 5354 -j ACCEPT
+# iptables -I INPUT -i in-$NAME -p tcp --destination $IPHOST --dport 9040 -j ACCEPT
+  iptables -I INPUT -i in-$NAME -p udp --destination $IPHOST --dport 9040 -j ACCEPT
```

Comment:
this should be reversed, the line with `-p udp` has to be removed not the one with `-p tcp`. correct? should you fix this?